### PR TITLE
Implemented and tested count_nonzero() function for Jax frontend

### LINKED ISSUE #22292: count_nonzero

### PR DESCRIPTION
**Pull request by lapatric**
Close #22292

### CODE REVIEW

**Comment by lapatric** [comment]
Could you please review when you have time, @hmahmood24?

File: ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_searching_sorting.py [hunk 1]
```diff
@@ -76,6 +76,36 @@ def test_jax_argwhere(
...
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        frontend=frontend,
+        backend_to_test=backend_fw,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        a=x[0],
+        axis=None,
+        keepdims=False,
```
Comment (hmahmood24) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_searching_sorting.py:105]: We are supposed to have strategies for testing the `axis` and `keepdims` arguments as well, just as we have for the `a` argument

**Review by hmahmood24 — CHANGES_REQUESTED** [review]
Hey @lapatric, requested changes for you to address. Also can you please fix the [lint/formatting](https://github.com/unifyai/ivy/actions/runs/5984134710/job/16235014138?pr=22442) issues with your files? Thanks! :)

**Comment by hmahmood24** [comment]
Closing this PR since there has been no activity for more than a week.

**Pull request closed without merging**